docs/tasks.rst

```diff
@@ -2,6 +2,19 @@
 Tasks Reference
 ==========================================
 
+assign_permission_set
+==========================================
+
+**Description:** Assign Permission Set specified in the "extra" option using [--permsetname "Permission_Set_DeveloperName"]
+
+**Class::** cumulusci.tasks.sfdx.SFDXBaseTask
+
+Options:
+------------------------------------------
+
+* **command** *(required)*: The full command to run with the sfdx cli. **Default: force:user:permset:assign**
+* **extra**: Append additional options to the command. **Specify Permission Set: --permsetname "Permission_Set_DeveloperName"**
```

Comment:
OK, one last minor change and this should be ready to merge.  The instructions here need to have `--permsetname` removed to reflect the changes in the cumulusci.yml.

Reply:
Thanks @jlantz , I've made your suggested change now! 👍